Libs/MRML/Widgets/qMRMLLayoutManager.cxx

```diff
@@ -1221,21 +1228,7 @@ void qMRMLLayoutManager::setRenderPaused(bool pause)
   foreach(const QString& viewName, sliceViewFactory->viewNodeNames())
     {
     ctkVTKAbstractView* view = this->sliceWidget(viewName)->sliceView();
-    if (!pause && !view->isRenderPaused())
-      {
-      // This view is already resumed and a resume is requested.
-      // This is probably because the view has just been added
-      // and so it missed a few PauseRender requests.
-      // Do not try to resume render to avoid logging of a warning,
-      // and just log a debug message here to help with troubleshooting if
-      // any problem comes up related to pausing rendering.
-      qDebug() << Q_FUNC_INFO << "Resume render request is ignored for view "
-        << viewName << ", probably the view has just been created";
-      }
-    else
-      {
-      view->setRenderPaused(pause);
-      }
+    view->setRenderPaused(pause);
```

Comment:
Please still keep the warning message if trying to unpause render on a view that has no paused rendering on it. Thanks to your fixes, we should no longer see the message.

Reply:
This isn't the warning message that we are used to seeing, This is a debug message that was hiding the warning message for slice views. The actual warning message is still in CTK: https://github.com/commontk/CTK/blob/master/Libs/Visualization/VTK/Widgets/ctkVTKAbstractView.cpp#L285.

Reply:
It is very good then, thanks for the clarification.